main.tex

```diff
@@ -16,8 +16,36 @@
 }
 
 \begin{document}
+\section{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen, facebook inc, stetig gewachsen. 
+Doch da die jährliche Anzahl neuer Kabelanschlüsse, für das Internet, global gesehen seit 2008 immer weniger geworden ist,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook äquivalent dazu auch nachgelassen.\\
+Dies passiert obwohl "at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in den Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
 
-\maketitle
+\medskip
 
+Daraus lässt sich ableiten, dass für den weiteren Wachstum des Internets, und der Dienstleistungen auf ihm (wie zum Beispiel Facebook), es nötig ist dass das Internet in den Entwicklungsländern ausgebreitet, und die Interesse der Menschen dort dafür geweckt wird. 
+
+\medskip
+
+Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat in 2013 das Dokument "Is Connectivity a Human Right?" veröffentlicht.
+In diesem erläutert er, dass es für Facebook wahrscheinlich nicht gewinnbringend sei, das Internet in den Entwicklungsländern auszubreiten. Trotzdem arbeitet Facebook daran dies zu machen, da er daran glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.(Vgl. \cite{HumanRight})
+
+\medskip
+
+Positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der 3. Welt, sind die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
```

Comment:
Quelle? Denn wenn das vom Zucc so behauptet wird, dann müsste das aus dem Text hervor gehen (z.b. Vermeintliche positive Effekte sind...)

Reply:
`3. Welt` als `dritte Welt` oder wenn man den Begriff vermeiden will z.B. als `Länder ohne weitdeckende Internetverbreitung`

Reply:
Also ich war mir bei den Zitaten/Quellen die ganze Zeit sehr unsicher.
Ich wollte natürlich nicht, dass mein ganzer Text nur aus Zitaten besteht, aber ich muss mich ja eigentlich ununterbrochen auf das Beziehen was in den Dokumenten steht.
Ich dachte hier wäre es in Ordnung, das einfach im Raum stehen zu lassen, da sich 
diese Aussage ja aus dem nächsten Zitat ableiten lässt.

vielleicht wäre es eine Idee den Absatz mit den vorherigen zu verbinden.
Also `da er daran glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.` 
und dann
`warum das verknüpft sein überhaupt was ist was man haben möchte => Auswirkungen`

Reply:
Das verbinden der Absätze kannst du probieren!

Und es muss auch nicht an jedem Absatz eine Quelle dranstehen, aber wenn es nicht deine eigene Meinung ist, dann sollte das halt aus dem Text hervor gehen :smile:

Reply:
Ja, stimmt.
Das macht Sinn, danke.